config/manager/manager.yaml

```diff
@@ -71,6 +71,7 @@ spec:
         args:
         - --leader-elect
         image: controller:latest
+        imagePullPolicy: Never
```

Comment:
Suggest use full image path `ghcr.io/aenix-io/controller:latest` and `imagePullPolicy: IfNotPresent` for reuse yaml in static manifests and helm.

Reply:
It should be `ghcr.io/aenix-io/etcd-operator/controller:latest` or `ghcr.io/aenix-io/etcd-operator:latest`

Reply:
Or `ghcr.io/aenix-io/etcd-operator/etcd-operator:latest`

@hiddenmarten could you comment on this, please?

Reply:
@kvaps 
Sure, I think it's better to use a simple structure of image tag, that will correspond with a GitHub repo path (It helps to avoid confusion):
`{registry}/{organisation}/{application}:{versionBySemVer}`

So, as a result:
`ghcr.io/aenix-io/etcd-operator:v0.0.0`